public/assets/css/rules.css

```diff
@@ -0,0 +1,230 @@
+h1,
```

Comment:
If h1 and h2 have this behavior in all website, please put it inside the global css file. Else it's fine